src/App.jsx

```diff
@@ -40,9 +40,16 @@ function App() {
       code: userCode,
       lang: userLang,
       input: userInput,
-    });
-    setUserOutput(response.data.stderr || response.data.stdout);
-    setIsLoading(false);
+    })
+    .then(() => {
```

Comment:
accept response in parameter i.e. it should be like .then((reponse))=>{})

Reply:
Have made the changes according to the comment.